WeakAurasOptions/AuthorOptions.lua

```diff
@@ -704,6 +705,178 @@ local typeControlAdders = {
     widthOption.order = order
     order = order + 1
 
+    return order
+  end,
+  multiselect = function(option, args, data, order, i)
+    args["option" .. i .. "width"] = nil
+    local values
+    local conflict = {} -- magic value
+    if option.references then
+      values = {}
+      for childID, optionID in pairs(option.references) do
+        local childData = data[childID]
+        local childValues = childData.authorOptions[optionID].values
+        local i = 1
+        while i <= #values or i <= #childValues do
+          local value = values[i]
+          if value == conflict then
+            -- conflicts can't ever be resolved at this point
+          elseif value == nil then
+            -- set the new value
+            values[i] = childValues[i]
+          elseif value ~= childValues[i] then
+            -- either this child has a conflicting value at this index
+            -- or it's already ended and is nil, so we need to mark a conflict
+            values[i] = conflict
+          end
+          i = i + 1
+        end
+      end
+    else
+      values = option.values
+    end
+    local defaultValues = {}
+    for i, v in ipairs(values) do
+      if v == conflict then
+        defaultValues[i] = "|cFF4080FF" .. L["Value %i"]:format(i)
+      else
+        defaultValues[i] = v
+      end
+    end
+    args["option" .. i .. "default"] = {
+      type = "multiselect",
+      width = WeakAuras.normalWidth,
+      name = L["Default"],
+      -- name = name(data, option, "default", L["Default"]),
+      -- desc = desc(data, option, "default"),
+      order = order,
+      values = defaultValues,
+      get = function(_, k)
+        return option.default and option.default[k]
+      end,
+      set = function(_, k, v)
+        if option.references then
+          for childID, optionID in pairs(option.references) do
+            local childData = data[childID]
+            local childOption = childData.authorOptions[optionID]
+            childOption.default[k] = v
```

Comment:
You might want to add a check here that the key is correct for that child option

I guess it's not needed since it'll be removed again by the validate.